plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/github/GHService.java

```diff
@@ -142,15 +142,17 @@ private void createAndCheckoutBranch(Path pluginDirectory, String branchName) th
     }
 
     public void commitAndCreatePR(String pluginName, String branchName) throws IOException, GitAPIException {
+        Path pluginDirectory = Paths.get(Settings.TEST_PLUGINS_DIRECTORY, pluginName);
+
         if (config.isDryRun()) {
             LOG.info("Skipping commit and pull request creation for {}", pluginName);
+            Path patchFilePath = pluginDirectory.resolve("target/rewrite/rewrite.patch");
+            LOG.info("Patch file path: {}", patchFilePath);
```

Comment:
I'm wonder if info is the good level (maybe debug ?) or if it should be save on the logs/<plugin>.log instead